### DISCUSSION TITLE
In the Flow (Intro to Python & Vyper)

### DISCUSSION THREAD

**Discussion by shubhamsinha21** [Cyfrin/moccasin-full-course-cu#13] [Show and tell]
Hey, I'm excited🔥to start this course today. Lets see how it goes 👋🏻

**Comment by Scofield-Idehen** [comment]
Best of luck!!!

**Reply by shubhamsinha21** [reply]
Thanks @Scofield-Idehen

**Comment by shubhamsinha21** [comment]
🔥I’ve completed 24% of the 6-hour Introduction to Python and Vyper course on Cyfrin Updraft. @PatrickAlphaC 

LinkedIn Post - https://www.linkedin.com/posts/shubham-sinha-39b9b2209_learningjourney-python-ai-activity-7275182538451496961-LLJn?utm_source=share&utm_medium=member_desktop

![FRESH](https://github.com/user-attachments/assets/cd5e4c59-a372-4642-b5b3-2239551c3814)

**Reply by PatrickAlphaC** [reply]
awesome!!!

**Comment by Michael-Lyon** [comment]
Hello People.
I'm Michael Asomugha, and I'm glad to be here.

I'm learning blockchain to see how i can build an ecosystem around the School Now Pay Later platform we're working on at https://payskul.com

**Reply by EngrPips** [reply]
Interesting.
